### PR TITLE
User guides updates and fixes: schema changes and Online DDL

### LINKED ISSUE #1238: [Docs Review] Making Schema Changes
Parent Issue vitessio/website#1222 
Vitess github url: https://github.com/vitessio/website

### Pages to review:

- [x] [Making Schema Changes](https://vitess.io/docs/15.0/user-guides/schema-changes/)
- [x] [Unmanaged Schema Changes](https://vitess.io/docs/16.0/user-guides/schema-changes/unmanaged-schema-changes/)
- [x] [Managed, Online Schema Changes](https://vitess.io/docs/16.0/user-guides/schema-changes/managed-online-schema-changes/)
- [x] [Online DDL strategies](https://vitess.io/docs/16.0/user-guides/schema-changes/ddl-strategies/)
- [x] [ddl_strategy flags](https://vitess.io/docs/16.0/user-guides/schema-changes/ddl-strategy-flags/)
- [x] [Table lifecycle](https://vitess.io/docs/16.0/user-guides/schema-changes/table-lifecycle/)
- [x] [Applying, auditing, and controlling Online DDL](https://vitess.io/docs/16.0/user-guides/schema-changes/audit-and-control/)
- [x] [Declarative migrations](https://vitess.io/docs/16.0/user-guides/schema-changes/declarative-migrations/)
- [x] [Postponed migrations](https://vitess.io/docs/16.0/user-guides/schema-changes/postponed-migrations/)
- [x] [Recoverable, failover agnostic migrations](https://vitess.io/docs/16.0/user-guides/schema-changes/recoverable-migrations/)
- [x] [Revertible migrations](https://vitess.io/docs/16.0/user-guides/schema-changes/revertible-migrations/)
- [x] [Concurrent migration execution](https://vitess.io/docs/16.0/user-guides/schema-changes/concurrent-migrations/)
- [x] [Advanced usage](https://vitess.io/docs/16.0/user-guides/schema-changes/advanced-usage/)

### What to do 

- [x] Make sure the content reflect the current state and facts are up-to-date.
- [x] Add details if needed either in form of new section on current page or adding new page.
- [x] Add links/Urls on page are working
- [x] Deprecate flags should be marked as 'deprecated'
- [ ] Review is done for both V15 and V16

Please note that we are not going to focus of restructuring the website content as part of this project. It is something we will do as a separate project later on.

### PR DESCRIPTION
**Pull request by shlomi-noach**
An ongoing PR that updates all documents around schema changes and Online DDL.

Closes https://github.com/vitessio/website/issues/1238

### CODE REVIEW

File: content/en/docs/16.0/user-guides/schema-changes/advanced-usage.md [hunk 1]
```diff
@@ -74,6 +74,40 @@ Notes:
...
+
+A `show vitess_migrations like '29231906_776f_11ec_a656_0a43f95f28a3'` presents the column `ready_to_complete`, with values `0` (not ready) or `1` (ready).
+
+When all migrations for the relevant UUIDs show `1` for `ready_to_complete`, the user can then either:
+
+```sh
+$ vtctlclient -- ApplySchema --skip_preflight --sql "alter vitess_migration complete all" commerce
+```
+
+assuming these are the only migrations swaiting completion, or, explicitly issue a complete for each of the migrations:
```
Comment (rsajwani) [inline content/en/docs/16.0/user-guides/schema-changes/advanced-usage.md]: swaiting ?? or waiting

File: content/en/docs/16.0/user-guides/schema-changes/ddl-strategies.md [hunk 2]
```diff
@@ -6,10 +6,10 @@ aliases: ['/docs/user-guides/schema-changes/ddl-strategies/']
...
 
-- `direct`: the direct apply of DDL to your database. This is not an online DDL. It is a synchronous and blocking operation. This is the default strategy. 
-- `vitess` (formerly known as `online`): utilizes Vitess's built in [VReplication](../../../reference/vreplication/vreplication/) mechanism. This is the preferred strategy in Vitess.
+- `vitess` (formerly known as `online`): utilizes Vitess's built-in [VReplication](../../../reference/vreplication/vreplication/) mechanism. This is the preferred strategy in Vitess.
 - `gh-ost`: uses 3rd party GitHub's [gh-ost](https://github.com/github/gh-ost) tool.
 - `pt-osc`: uses 3rd party Percona's [pt-online-schema-change](https://www.percona.com/doc/percona-toolkit/3.0/pt-online-schema-change.html) as part of [Percona Toolkit](https://www.percona.com/doc/percona-toolkit/3.0/index.html). `pt-osc` strategy is **experimental**.
+- `direct`: the direct apply of DDL to your database. This is not an online DDL. It is a synchronous and blocking operation. This is the default strategy.
```
Comment (rsajwani) [inline content/en/docs/16.0/user-guides/schema-changes/ddl-strategies.md]: for my information : is direct == unmanaged?

File: content/en/docs/16.0/user-guides/schema-changes/managed-online-schema-changes.md [hunk 3]
```diff
@@ -219,7 +221,7 @@ All three strategies: `vitess`, `gh-ost` and `pt-osc` utilize the tablet throttl
 All `ALTER` strategies leave artifacts behind. Whether successful or failed, either the original table or the _ghost_ table is left still populated at the end of the migration. Vitess explicitly makes sure the tables are not dropped at the end of the migration. This is for two reasons:
 
 - Make the table/data still available for a while, and
-- in MySQL pre `8.0.23`, a `DROP TABLE` operation can be dangerous in production as it commonly locks the buffer pool for a substantial period.
+- In MySQL pre `8.0.23`, a `DROP TABLE` operation can be dangerous in production as it commonly locks the buffer pool for a substantial period.
```
Comment (rsajwani) [inline content/en/docs/16.0/user-guides/schema-changes/managed-online-schema-changes.md]: minor: prior to MySQL 8.0.23

**Review by rsajwani — APPROVED** [review]
some minor comments

File: content/en/docs/16.0/user-guides/schema-changes/advanced-usage.md (hunk 1 above)
Comment (shlomi-noach) [reply content/en/docs/16.0/user-guides/schema-changes/advanced-usage.md]: fixed

File: content/en/docs/16.0/user-guides/schema-changes/ddl-strategies.md (hunk 2 above)
Comment (shlomi-noach) [reply content/en/docs/16.0/user-guides/schema-changes/ddl-strategies.md]: correct! Also clarified in text.

File: content/en/docs/16.0/user-guides/schema-changes/managed-online-schema-changes.md (hunk 3 above)
Comment (shlomi-noach) [reply content/en/docs/16.0/user-guides/schema-changes/managed-online-schema-changes.md]: Fixed! Please note this PR is still in draft, there's more changes coming today.

**Pull request merged**